src/lib/telemetry/TelemetryReporter.ts

```diff
@@ -117,16 +139,39 @@ export class TelemetryReporter {
       }
       return error
     })
+    await ensureDir(join(TelemetryReporter.PENDING_DATA_DIR, 'errors'))
```

Comment:
Maybe this `join` could be an static variable, what do you think? `static private PENDING_ERRORS_DIR`?